proto/regen/ecocredit/v1/tx.proto

```diff
@@ -421,3 +427,21 @@ message MsgUpdateProjectMetadata {
 // MsgUpdateProjectMetadataResponse is the Msg/UpdateProjectMetadataResponse
 // response type.
 message MsgUpdateProjectMetadataResponse {}
+
+// MsgBridge is the Msg/Bridge request type.
+message MsgBridge {
+  // MsgCancel is the MsgCancel message type.
+  MsgCancel msg_cancel = 1;
+
+  // bridge_target is the target chain.
+  string bridge_target = 2;
+
+  // bridge_recipient is the recipient address.
+  string bridge_recipient = 3;
+
+  // bridge_contract is the contract address.
+  string bridge_contract = 4;
```

Comment:
Same as above, I think we should exclude the `bridge_` prefix on these. cc @clevinson 

```suggestion
  // target is the target chain.
  string target = 2;

  // recipient is the recipient address.
  string recipient = 3;

  // contract is the contract address.
  string contract = 4;
```